### PR TITLE
Added FieldPath fuzzing target

### PR DESCRIPTION
**Pull request by minafarid**

### CODE REVIEW

File: Firestore/core/src/firebase/firestore/fuzzing/fieldpath_fuzzer.cc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+#include <cstdint>
+
+#include "Firestore/core/src/firebase/firestore/model/field_path.h"
+
+namespace firebase {
+namespace firestore {
+namespace fuzzing {
+namespace fieldpath {
+
+using firebase::firestore::model::FieldPath;
```
[the remark this answers is not in the document]
Comment (minafarid) [reply Firestore/core/src/firebase/firestore/fuzzing/fieldpath_fuzzer.cc]: Ok.

File: Firestore/core/src/firebase/firestore/fuzzing/serializer_fuzzer.cc [hunk 2]
```diff
@@ -20,10 +20,15 @@
 #include "Firestore/core/src/firebase/firestore/model/database_id.h"
 #include "Firestore/core/src/firebase/firestore/remote/serializer.h"
 
+namespace firebase {
+namespace firestore {
+namespace fuzzing {
+namespace serializer {
```
[the remark this answers is not in the document]
Comment (minafarid) [reply Firestore/core/src/firebase/firestore/fuzzing/serializer_fuzzer.cc]: Noted, removed namespaces. Thanks.

File: Firestore/core/src/firebase/firestore/fuzzing/CMakeLists.txt [hunk 3]
```diff
@@ -35,16 +39,33 @@ cc_binary(
...
 )
 
-# Copy the serializer dictionary and corpus from the FuzzingResources in iOS
-# fuzz tests.
+# Copy serializer dictionary and corpus.
 # TODO(minafarid): Generate the serializer corpus by converting the text protos
 # from the serializer corpus in the iOS FuzzingResources to binary protos. This
 # conversion requires the protoc binary that is not currently available.
-set(serializer_resources ${FIREBASE_SOURCE_DIR}/Firestore/Example/FuzzTests/FuzzingResources/Serializer)
 add_custom_command(
```
[the remark this answers is not in the document]
Comment (rsgowman) [reply Firestore/core/src/firebase/firestore/fuzzing/CMakeLists.txt]: > cc_fuzz_test

Sounds like a good idea to me.

> in a follow-up

Also sounds like a good idea. :)

**Comment by minafarid** [comment]
Will work on creating `cc_fuzz_test` and refactoring what we have next.

**Pull request merged**